components/log-viewer-webui/client/src/main.tsx

```diff
@@ -1,19 +1,33 @@
 import {StrictMode} from "react";
 import {createRoot} from "react-dom/client";
 
+import AntdApp from "./AntdApp";
 import App from "./App";
 
 import "./index.css";
 
 
+/* eslint-disable-next-line no-warning-comments */
+// TODO: Remove flag and related logic when the new UI is fully implemented.
+const antdFlag = false;
```

Comment:
How about writing it as `const {USE_ANTD_APP} = import.meta.env;` and adding an npm script `"dev:antd": "USE_ANTD_APP=true npm run dev"`?

Reply:
And document the purpose script somewhere in a Markdown doc?